pages/common/detox.md

```diff
@@ -0,0 +1,16 @@
+# detox
+
+> Renames files to make them easier to work with. It removes spaces 
+> and other such annoyances like duplicate underline characters. 
+
+- Remove spaces and other undesirable characters from a file's name:
+
+`detox {{file}}`
+
+- Show how detox would rename all of the files in a directory tree:
```

Comment:
`-n` is for dry-run right ? How about - `Perform a dry-ru[n] of detox recursively on all files in a given directory`

Reply:
Could use the full option of `--dry-run`?

Reply:
Well, sure. That's why I highlighted the `[n]` in dry-run so that it's understood. But I am okay with both approaches.